### DISCUSSION TITLE
Fine grained ACL for api-token

### DISCUSSION THREAD

**Discussion by psytron** [nocodb/nocodb#3544] [General]
Simple use case:  Visitors sign up for mailing list on website. 

Web form works with API token, BUT same API token can expose entire database.... 

How can we have API token for web forms, write-only, and not expose entire tables?

**Comment by o1lab** [comment]
Better use here would be to use our forms as an iframe and embed in your app. Moving to discussion.

**Comment by psytron** [comment]
Embedding iFrame doesn't allow wide range of data sources, and makes customizing form difficult.  This would easily be fixed with fine grained ACL for API-tokens.  Simply have write-only function for web forms or other inputs.
